### ISSUE TITLE
Automatically add comment to all pull requests for an issue when another PR is filed

### ISSUE THREAD

**Issue by jenlampton** [backdrop-ops/backdropcms.org#12]
We'll need to let the authors of pull requests know when newer (probably more relevant) PRs have been created. 

As well as updating the original issue, it would be nice if we could add a comment to previous PRs notifying them that they are now out of date.

"@whoever has created a new pull request for this issue: link"

Similar to https://github.com/backdrop/backdropcms.org-issues/issues/11

**Comment by Gormartsen** [comment]
Yes it is possible to add comment to different PR related to the same issue.
When cross linking for backdrop-issues and backdrop repo will be production, it will be possible to find all related PR via events api and add comment to PR
